blocksuite/affine/shared/src/services/font-loader/font-loader-service.ts

```
@@ -20,33 +20,105 @@ const initFontFace = IS_FIREFOX
+    if (!config || config.length === 0) {
+      return;
     }
+    const eagerFonts = config.slice(0, 3);
+    const deferredFonts = config.slice(3);
+    this.load(eagerFonts);
+    this._scheduleDeferredLoad(deferredFonts);
   }
 
   override unmounted() {
+    this._cancelDeferredLoad();
     this.fontFaces.forEach(fontFace => document.fonts.delete(fontFace));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)